code-challenges.test.js

```diff
@@ -44,9 +81,30 @@ var fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
 // b) Create the function that makes the test pass.
 
 
-
+describe("The onlyOdds function.",()=>{
+    test("takes in an array and returns a new array of only odd numbers sorted from least to greatest", () => {
+        expect(onlyOdds(fullArr1)).toEqual([-9, 7, 9, 199])
+        expect(onlyOdds(fullArr2)).toEqual([-823, 7, 23])
+    })
+})
+
+const onlyOdds = (array) =>{
+    let sortedArr = array.filter(function (item) {
+    if (typeof item === `number` && item%2)
```

Comment:
This totally works! but it's important to remember that .filter has a built in conditional statement. That's the reason we want to use filter instead of something like a for loop.